packages/typescript/src/typescript.ts

```diff
@@ -60,7 +60,11 @@ export function typescript(configFilePath: string, extraCompilerOptions: ts.Comp
     ]);
 
     return allDiagnostics.reduce((fileInfoMap, diagnostic) => {
-      const { file, start, length } = diagnostic as ts.DiagnosticWithLocation;
+      const { file, start, length } = diagnostic;
+      // If there is no location, skip it
+      if (file === undefined || start === undefined || length === undefined) {
+        return fileInfoMap;
+      }
```

Comment:
If you don't mind I'd probably change this to be a `filter` before the `reduce`?

```suggestion
    return allDiagnostics
      .filter(({ file, start, length }) => file && start != null && length != null)
      .reduce(/* ... */);
```

Reply:
I am personally no fan of unnecessary casting (like as `ts.DiagnosticWithLocation`) because it weakens the strongness of a mighty typesystem. But those are just my 2 cents, of course I applied your suggestion. ☺️